dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+            });
+
+            it('Check getIds and return array', function () {
+
+                overlayObj.getIds = jasmine.createSpy().and.callFake(function () {
+                    return [1,2,3];
+                });
+
+                expect(overlayObj.getIds().length).toBe(3);
+            });
+
+        });
+
+        describe('"getStyles" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('getStyles')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.getStyles;
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value type if method called with object argument', function () {
+
+                overlayObj.getStyles = jasmine.createSpy().and.callFake(function () {
+                    return {
+                        top: '100px'
+                    };
+                });
+
+                expect(typeof overlayObj.getStyles()).toEqual('object');
+            });
+
+            it('Check getStyles and return array', function () {
+
+                overlayObj.getStyles = jasmine.createSpy().and.callFake(function () {
+                    return {
+                        top: '100px'
+                    };
+                });
+
+                expect(overlayObj.getStyles().top).toBe('100px');
+            });
+
+        });
+
+        describe('"isVisible" method', function () {
+
+            it('Check for defined', function () {
+                expect(overlayObj.hasOwnProperty('isVisible')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.isVisible;
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value type if method called with object argument', function () {
+                var row = {
+                    id: 123213
+                };
+
+                overlayObj.isVisible = jasmine.createSpy().and.callFake(function () {
+                    return true;
+                });
+
+                expect(typeof overlayObj.isVisible(row)).toEqual('boolean');
+            });
+
+            it('Check called "this.isVisible" method with object argument', function () {
+                var row = {
+                    id: 123213
+                };
+                overlayObj.isVisible = jasmine.createSpy().and.callFake(function () {
+                    return true;
+                });
+                overlayObj.isVisible(row);
+                expect(overlayObj.isVisible).toHaveBeenCalledWith(row);
+            });
+
+            it('Check isVisible and return false if id doesnt found', function () {
+                var row = {
+                    id: ''
+                };
+                overlayObj.isVisible = jasmine.createSpy().and.callFake(function () {
```

Comment:
This test is not valuable as you are stubbing out the `isVisible` directly before testing it. Either don't stub the method you are testing, or remove this test.